app/javascript/react/src/components/App.jsx

```diff
@@ -1,15 +1,25 @@
-import React from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import Greeting from './Greeting';
-
-function App() {
-  return (
-    <Router>
-      <Routes>
-        <Route path="/" element={<Greeting />} />
-      </Routes>
-    </Router>
-  );
-}
-
-export default App;
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Greeting from './Greeting';
+
+function App() {
+  return (
+      <Routes>
+        <Route path="/" element={<Greeting />} />
+      </Routes>
+  );
+};
+
+const root = ReactDOM.createRoot(document.getElementById('root'))
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <Router>
+        <App />
+      </Router>
+    </Provider>
+  </React.StrictMode>
+);
+
+
+export default App;

```

Comment:
- [ ] @Kingno3l Great work implementing the changes requested, however the app is not working i am getting the output below, kindly consider removing the extra code in your root view to only have `<div id="root">`, that connects to your react app, also you have not modified the `javascript/application.js` as previously requested, this file serves as the entry point for the build script in the package.json, I will repost the suggested code below
![image](https://github.com/Kingno3l/React-on-Rails/assets/61408860/01efa8a2-33b3-4d18-b426-d3968753d28c)
![image](https://github.com/Kingno3l/React-on-Rails/assets/61408860/5870cea5-211f-4c1d-b521-845cb46c163d)